utilities/generate_api.py

```diff
@@ -30,7 +30,6 @@
 
 
 dir_overrides = {
-    'bigtable-admin': 'google-cloud-bigtable',
     'error-reporting': 'google-cloud-errorreporting',
```

Comment:
Could you add bigtable-admin to the batch generation script?

Reply:
It's already there. This client was already being generated, but was copied to a different directory

Reply:
Oh right! Sounds good.